kubernetes-client/src/main/java/io/fabric8/kubernetes/client/Config.java

```diff
@@ -68,6 +68,8 @@
   public static final String KUBERNETES_ALL_PROXY = "all.proxy";
   public static final String KUBERNETES_NO_PROXY = "no.proxy";
 
+  public static final String KUBERNETES_USER_AGENT = "fabric8-kubernetes-client";
```

Comment:
Added version available as `Version.clientVersion()` so can we add the default user agent as `"fabric8-kubernetes-client/" + Version.clientVersion()`.